File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/phys/Phy8002U01GrdPhy8004LisItemInfo.java
package nta.med.data.model.ihis.phys;

import java.util.Date;

public class Phy8002U01GrdPhy8004LisItemInfo {
	private Date sysDate;
	private String userId;
	private Date updDate;
	private String hospCode;
	private Double pkPhySyougai;
	private String dataKubun;
	private Double fkOcsIrai;
	private String syougaiId;
	private String syougaimei;
	private String kanjaNo;
	private Double fkcht0113;

	public Phy8002U01GrdPhy8004LisItemInfo(Date sysDate, String userId,
			Date updDate, String hospCode, Double pkPhySyougai,
			String dataKubun, Double fkOcsIrai, String syougaiId,
			String syougaimei, String kanjaNo, Double fkcht0113) {
		super();
		this.sysDate = sysDate;
		this.userId = userId;
		this.updDate = updDate;
		this.hospCode = hospCode;
		this.pkPhySyougai = pkPhySyougai;
		this.dataKubun = dataKubun;
		this.fkOcsIrai = fkOcsIrai;
		this.syougaiId = syougaiId;
		this.syougaimei = syougaimei;
		this.kanjaNo = kanjaNo;
		this.fkcht0113 = fkcht0113;
	}

	public Date getSysDate() {
		return sysDate;
	}

	public void setSysDate(Date sysDate) {
		this.sysDate = sysDate;
	}

	public String getUserId() {
		return userId;
	}

	public void setUserId(String userId) {
		this.userId = userId;
	}

	public Date getUpdDate() {
		return updDate;
	}

	public void setUpdDate(Date updDate) {
		this.updDate = updDate;
	}

	public String getHospCode() {
		return hospCode;
	}

	public void setHospCode(String hospCode) {
		this.hospCode = hospCode;
	}

	public Double getPkPhySyougai() {
		return pkPhySyougai;
	}

	public void setPkPhySyougai(Double pkPhySyougai) {
		this.pkPhySyougai = pkPhySyougai;
	}

	public String getDataKubun() {
		return dataKubun;
	}

	public void setDataKubun(String dataKubun) {
		this.dataKubun = dataKubun;
	}

	public Double getFkOcsIrai() {
		return fkOcsIrai;
	}

	public void setFkOcsIrai(Double fkOcsIrai) {
		this.fkOcsIrai = fkOcsIrai;
	}

	public String getSyougaiId() {
		return syougaiId;
	}

	public void setSyougaiId(String syougaiId) {
		this.syougaiId = syougaiId;
	}

	public String getSyougaimei() {
		return syougaimei;
	}

	public void setSyougaimei(String syougaimei) {
		this.syougaimei = syougaimei;
	}

	public String getKanjaNo() {
		return kanjaNo;
	}

	public void setKanjaNo(String kanjaNo) {
		this.kanjaNo = kanjaNo;
	}

	public Double getFkcht0113() {
		return fkcht0113;
	}

	public void setFkcht0113(Double fkcht0113) {
		this.fkcht0113 = fkcht0113;
	}

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Phys/PHY8002U01GrdPHY8004LisItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Phys
{
    [Serializable]
    public class PHY8002U01GrdPHY8004LisItemInfo
    {
        private String _sysDate;
        private String _userId;
        private String _updDate;
        private String _hospCode;
        private String _pkPhySyougai;
        private String _dataKubun;
        private String _fkOcsIrai;
        private String _syougaiId;
        private String _syougaimei;
        private String _kanjaNo;
        private String _fkcht0113;
        private String _rowState;

        public String SysDate
        {
            get { return this._sysDate; }
            set { this._sysDate = value; }
        }

        public String UserId
        {
            get { return this._userId; }
            set { this._userId = value; }
        }

        public String UpdDate
        {
            get { return this._updDate; }
            set { this._updDate = value; }
        }

        public String HospCode
        {
            get { return this._hospCode; }
            set { this._hospCode = value; }
        }

        public String PkPhySyougai
        {
            get { return this._pkPhySyougai; }
            set { this._pkPhySyougai = value; }
        }

        public String DataKubun
        {
            get { return this._dataKubun; }
            set { this._dataKubun = value; }
        }

        public String FkOcsIrai
        {
            get { return this._fkOcsIrai; }
            set { this._fkOcsIrai = value; }
        }

        public String SyougaiId
        {
            get { return this._syougaiId; }
            set { this._syougaiId = value; }
        }

        public String Syougaimei
        {
            get { return this._syougaimei; }
            set { this._syougaimei = value; }
        }

        public String KanjaNo
        {
            get { return this._kanjaNo; }
            set { this._kanjaNo = value; }
        }

        public String Fkcht0113
        {
            get { return this._fkcht0113; }
            set { this._fkcht0113 = value; }
        }

        public String RowState
        {
            get { return this._rowState; }
            set { this._rowState = value; }
        }

        public PHY8002U01GrdPHY8004LisItemInfo() { }

        public PHY8002U01GrdPHY8004LisItemInfo(String sysDate, String userId, String updDate, String hospCode, String pkPhySyougai, String dataKubun, String fkOcsIrai, String syougaiId, String syougaimei, String kanjaNo, String fkcht0113, String rowState)
        {
            this._sysDate = sysDate;
            this._userId = userId;
            this._updDate = updDate;
            this._hospCode = hospCode;
            this._pkPhySyougai = pkPhySyougai;
            this._dataKubun = dataKubun;
            this._fkOcsIrai = fkOcsIrai;
            this._syougaiId = syougaiId;
            this._syougaimei = syougaimei;
            this._kanjaNo = kanjaNo;
            this._fkcht0113 = fkcht0113;
            this._rowState = rowState;
        }

    }
}